run_unittests.py

```diff
@@ -5658,9 +5658,10 @@ def test_clang_format(self):
             output = self.build('clang-format-check')
             self.assertEqual(1, output.count('File reformatted:'))
 
-        # All code has been reformatted already, so it should be no-op now.
+        # The check format should not touch any files. Thus
+        # running format again has some work to do.
         output = self.build('clang-format')
-        self.assertEqual(0, output.count('File reformatted:'))
+        self.assertEqual(1, output.count('File reformatted:'))
         self.build('clang-format-check')
```

Comment:
nitpicking, could keep the comment `# All code has been reformatted already, so it should be no-op now.` and move it here.